superset-frontend/src/theme/light.ts

```diff
@@ -0,0 +1,119 @@
+    colorInfoBgHover: supersetTheme.colors.info.light1,
+    colorInfoBorder: supersetTheme.colors.info.light1,
+    colorInfoBorderHover: supersetTheme.colors.info.dark1,
+    colorInfoHover: supersetTheme.colors.info.dark1,
+    colorInfoText: supersetTheme.colors.info.dark1,
+    colorInfoTextActive: supersetTheme.colors.info.dark2,
+    colorInfoTextHover: supersetTheme.colors.info.dark1,
+
+    colorLinkActive: supersetTheme.colors.info.dark2,
+    colorLinkHover: supersetTheme.colors.info.dark1,
+
+    colorPrimaryActive: supersetTheme.colors.primary.dark2,
+    colorPrimaryBg: supersetTheme.colors.primary.light4,
+    colorPrimaryBgHover: supersetTheme.colors.primary.light3,
+    colorPrimaryBorder: supersetTheme.colors.primary.light2,
+    colorPrimaryBorderHover: supersetTheme.colors.primary.light1,
+    colorPrimaryHover: supersetTheme.colors.primary.dark1,
+    colorPrimaryText: supersetTheme.colors.primary.dark1,
+    colorPrimaryTextActive: supersetTheme.colors.primary.dark2,
+    colorPrimaryTextHover: supersetTheme.colors.primary.dark1,
+
+    colorSuccessActive: supersetTheme.colors.success.dark1,
+    colorSuccessBg: supersetTheme.colors.success.light2,
+    colorSuccessBgHover: supersetTheme.colors.success.light1,
+    colorSuccessBorder: supersetTheme.colors.success.light1,
+    colorSuccessBorderHover: supersetTheme.colors.success.dark1,
+    colorSuccessHover: supersetTheme.colors.success.dark1,
+    colorSuccessText: supersetTheme.colors.success.dark1,
+    colorSuccessTextActive: supersetTheme.colors.success.dark2,
+    colorSuccessTextHover: supersetTheme.colors.success.dark1,
+
+    colorText: supersetTheme.colors.grayscale.dark2,
+    colorTextQuaternary: supersetTheme.colors.grayscale.light1,
+    colorTextSecondary: supersetTheme.colors.text.label,
+    colorTextTertiary: supersetTheme.colors.text.help,
+
+    colorWarningActive: supersetTheme.colors.warning.dark1,
+    colorWarningBg: supersetTheme.colors.warning.light2,
+    colorWarningBgHover: supersetTheme.colors.warning.light1,
+    colorWarningBorder: supersetTheme.colors.warning.light1,
+    colorWarningBorderHover: supersetTheme.colors.warning.dark1,
+    colorWarningHover: supersetTheme.colors.warning.dark1,
+    colorWarningText: supersetTheme.colors.warning.dark1,
+    colorWarningTextActive: supersetTheme.colors.warning.dark2,
+    colorWarningTextHover: supersetTheme.colors.warning.dark1,
+
+    colorWhite: supersetTheme.colors.grayscale.light5,
+
+    fontSizeHeading1: supersetTheme.typography.sizes.xxl,
+    fontSizeHeading2: supersetTheme.typography.sizes.xl,
+    fontSizeHeading3: supersetTheme.typography.sizes.l,
+    fontSizeHeading4: supersetTheme.typography.sizes.m,
+    fontSizeHeading5: supersetTheme.typography.sizes.s,
+
+    fontSizeLG: supersetTheme.typography.sizes.l,
+    fontSizeSM: supersetTheme.typography.sizes.s,
+    fontSizeXL: supersetTheme.typography.sizes.xl,
+
+    lineWidthBold: supersetTheme.gridUnit / 2,
```

Comment:
Nit: do we want to tie this to the grid unit?

Reply:
We can revisit this as we continue the upgrade. The general theme is the result of the usage of the individual components, so this needs to be thought of as a draft for now.